frontend/src/App.tsx

```diff
@@ -61,8 +65,11 @@ function App(props: { setOnDisconnect: Dispatch<SetStateAction<Callback | undefi
       return <div>Loading...</div>;
     }
     return (
-      <div>
-        <div className='world-and-overlay-container'>
+      <div className='page-container'>
+        <div
+          onMouseEnter={removeFocusFromInputOrTabs}
+          onFocus={removeFocusFromInputOrTabs}
```

Comment:
you need `onFocus` to please the linter even though it's fundamentally useless here. keyboard users would translate focus from the chat no matter what to control the game.